ballsdex/packages/admin/info.py

```diff
@@ -129,6 +129,26 @@ async def user(
             catch_date__gte=datetime.datetime.now() - datetime.timedelta(days=days),
             player=player,
         )
+        class recentlyCaughtButton(discord.ui.View):
+            @discord.ui.button(label='Recent Catches', style = discord.ButtonStyle.primary)
+            async def on_button_click(self, interaction: discord.Interaction, button: discord.ui.Button) -> None:
+                await interaction.response.defer(ephemeral=True, thinking=True)
+                embed = discord.Embed(
+                    title=f"Catch times for {user.id} - last 30 catches",
+                )
+                recently_caught_balls = await BallInstance.filter(
+                    player=player,
+                ).order_by("-catch_date").limit(30)
+                ballInfo = ""
+                for ball in recently_caught_balls:
+                    catch_time = ball.catch_date - ball.spawned_time
+                    ballInfo = f"{ballInfo}{ball.description(short=True)} - {catch_time} - {ball.server_id}\n" 
```

Comment:
```suggestion
                    ballInfo += f"{ball.description(short=True)} - {catch_time} - {ball.server_id}\n" 
```

Instead of overwriting ballInfo each time, you can append to it